pytorch_toolkit/object_detection/model_templates/custom-detection/mobilenet_v2-2s_ssd-256x256/template.yaml

```diff
@@ -35,6 +34,12 @@ hyper_parameters:
     batch_size: 64
     base_learning_rate: 0.05
     epochs: 15
+  extra:
+    custom_classes:
+      name: "Custom classes"
+      param: "--classes"
+      type: "string"
+      default: ""
```

Comment:
Classes here are not the part of `dataset_requirements->classes` because those cannot be configured by the user, right? Or what is the reason?

Reply:
short: yes

dataset_requirements/classes is used to check dataset, whether it is ok for training or not.